docs/070-breaking-changes.rst

```diff
@@ -8,13 +8,103 @@ affected code.
 For the full list check
 `the release changelog <https://github.com/ethereum/solidity/releases/tag/v0.7.0>`_.
 
+
+Changes the Compiler Might not Warn About
+=========================================
+
+ * Exponentiation and shifts of literals by non-literals (e.g. ``1 << x`` or ``2 ** x``)
+   will always use either the type ``uint256`` (for non-negative literals) or
+   ``int256`` (for negative literals) to perform the operation in.
+   Previously, the operation was performed in the type of the shift amount / the
+   exponent which we think is misleading.
+
+
+Changes to the Syntax
+=====================
+
+ * In external function and contract creation calls, Ether and gas is now specified using a new syntax:
+   ``x.f{gas: 10000, value: 2 ether}(arg1, arg2)``.
+   The old syntax -- ``x.f.gas(10000).value(2 ether)(arg1, arg2)`` -- will cause an error.
+ * The global variable ``now`` is deprecated, ``block.timestamp`` should be used.
+   We think that ``now`` is too generic for a global variable and could given the impression
+   that it changes during transaction processing. ``block.timestamp`` correctly
+   reflects the fact that it is just a property of the block.
+ * NatSpec comments on variables are only allowed for public state variables and not
+   for local or internal variables.
+
+ * The token ``gwei`` is a keyword now (used to specify e.g. ``2 gwei`` as a number)
+   and cannot be used as an identifier.
```

Comment:
Somehow it feels awkward to start a sentence with a specially-formatted symbol. Maybe that's my latex education, but I would always try to avoid starting a sentence with a variable, for example.